google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/AppProfile.java

```diff
@@ -409,4 +412,108 @@ public int hashCode() {
       return Objects.hashCode(proto);
     }
   }
+
+  /** Compute Billing Owner specifies how usage should be accounted when using Data Boost. */
+  public static enum ComputeBillingOwner {
+    UNSPECIFIED(DataBoostIsolationReadOnly.ComputeBillingOwner.COMPUTE_BILLING_OWNER_UNSPECIFIED),
+    HOST_PAYS(DataBoostIsolationReadOnly.ComputeBillingOwner.HOST_PAYS);
+
```

Comment:
I think you need UNRECOGNIZED as well. (In case the proto is updated with new a variant that we dont know about